MonacoEditorComponent/Monaco/Helpers/CssStyleBroker.cs

```diff
@@ -31,24 +28,27 @@ static CssStyleBroker()
         private CssStyleBroker()
         {
         }
-        public static CssStyleBroker Instance // TODO: Probably need to tie this to a specific Editor
+        public static CssStyleBroker GetInstance(CodeEditor editor)
         {
-            get
-            {
-                return _instance;
-            }
+            if (instances.ContainsKey(editor)) return instances[editor];
```

Comment:
Might be better to use a Concurrent Dictionary here just to be safe, see example in the Windows Community Toolkit here: https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit/Helpers/Singleton.cs

Reply:
This is the biggest interface change that I haven't really dug into yet, so I need to try it out more.  I unfortunately haven't had a chance to finish playing with the NuGet package setup and tests before my busy period, so I may have to finish reviewing the PR first thing in March. :( It's really looking awesome though, and I appreciate all the work on this @hez2010.